packages/deployer/README.md

```diff
@@ -1,132 +1,65 @@
 # @api3/deployer
 
-> Deployment tools for Airnode
+> Airnode deployment CLI
 
-*The recommended way to deploy Airnode is by using the deployment Docker image.
-This package simply implements the scripts used by that image and is not meant to be used directly by the end user.*
+**The recommended way to deploy Airnode is by using the [Deployer Docker](../../docker/README.md) image. This package simply implements the CLI used by that image and is not meant to be used directly by the end user.**
 
-## Temporary instructions
-
-At the root
-```sh
-yarn run build-all
-```
-In this package
-```sh
-yarn run webpack
-yarn run webpack:copy-config
-yarn run command:deploy:nonstop
-yarn run command:remove-with-receipt
-```
-
-The information below is outdated.
+## Prerequisites
+* Install [Terraform v0.15.x](https://www.terraform.io/downloads.html) and make sure that the `terraform` binary is available in your `PATH`
+* Make sure your AWS credentials are stored in the [configuration file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-where) or exported as [environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html#envvars-set)
 
 ## Setup
-
-- Download the [Terraform v0.13.* binary](https://www.terraform.io/downloads.html) and move it to your `PATH`
-- Get an *Access Key ID* and *Secret Access Key* from AWS and insert into `.env` in the package directory (refer to `.env.example`)
-- Install Serverless Framework globally
-
-```sh
-npm install -g serverless
+* Build all the Airnode packages
+```bash
+# Run from the root
+yarn build
+```
+* Make sure `config.json` and `secrets.env` are available in the `config` directory
+```bash
+# From this package (/packages/deployer)
```

Comment:
I would add something like: "You can use the provided example `config.json` and `security.env` templates to get started quickly, but you will need to edit these with your own API details and secrets." just to make it clear that they don't **need** to copy the example files.